test/assignment.coffee

```diff
@@ -234,7 +234,71 @@ test "destructuring assignment against an expression", ->
   [y, z] = if true then [a, b] else [b, a]
   eq a, y
   eq b, z
-
+  
+test "destructuring assignment with objects and splats: ES2015", ->
+  obj = {a:1, b:2, c:3, d:4, e:5}
+  throws (-> CoffeeScript.compile "{a, r..., s....} = x"), null, "multiple rest elements are disallowed"
```

Comment:
I meant, it works in Babel so presumably it’s correct syntax. And therefore we should support the CoffeeScript equivalent (`{ a.b... }` instead of `{ ...a.b }`).